tools/import/import.js

```diff
@@ -177,6 +191,7 @@ function addJobsList(jobs) {
       downloadLink.download = 'import_results.zip';
       resultsContainer.append(downloadLink);
     }
+    displayStopButton(job.status === 'RUNNING');
```

Comment:
Wondering if for later, we could provide a Stop option regardless of whether or not a job was started from labs-ui?

Reply:
If you know the job id I think the labs UI will fetch te results of that job. What we need is an API to fetch all the jobs for an api-key.

Reply:
Maybe add all the api-key's jobs in the dropdown of jobs (in addition to the ones in localstorage)...
Some day.